test/functional/helpers.lua

```diff
@@ -411,7 +411,6 @@ end
 function module.clear(...)
   local argv, env = module.new_argv(...)
   module.set_session(module.spawn(argv, nil, env))
-  module.request('nvim_set_client_info', 'testclient', {}, 'remote', {}, {})
```

Comment:
Removing this line from the code as it causes the `encoding` test given at [test/functional/ex_cmds/encoding_spec.lua:24](https://github.com/neovim/neovim/blob/master/test/functional/ex_cmds/encoding_spec.lua#L24) to wait indefinitely.